### DISCUSSION TITLE
doriswriter编译失败

### DISCUSSION THREAD

**Discussion by ylpeng1122** [apache/doris#14659] [A - General / Q&A] [answered]
官网中提供的压缩包https://doris-thirdparty-repo.bj.bcebos.com/thirdparty/alibaba-datax-maven-m2-20210928.tar.gz没有datax-all的包，大佬们是怎么编译成功的？

**Comment by LemonLiTree** [comment, marked as answer]
直接用datax最新版就行了，已经直接支持了。
<img width="855" alt="image" src="https://user-images.githubusercontent.com/109049295/204429738-f88953f1-74fe-4ac5-8611-fc2f4af2ad68.png">

**Reply by ylpeng1122** [reply]
好的，感谢
